src/chainparams.cpp

```diff
@@ -15,10 +15,51 @@
 
 #include "base58.h"
 
-using namespace std;
-
 #include "chainparamsseeds.h"
 
+static CBlock CreateGenesisBlock(const char* pszTimestamp, const CScript& genesisOutputScript, uint32_t nTime, const uint256& nNonce, const std::vector<unsigned char>& nSolution, uint32_t nBits, int32_t nVersion, const CAmount& genesisReward)
+{
+    CMutableTransaction txNew;
+    txNew.nVersion = 1;
+    txNew.vin.resize(1);
+    txNew.vout.resize(1);
+    txNew.vin[0].scriptSig = CScript() << 520617983 << CScriptNum(4) << std::vector<unsigned char>((const unsigned char*)pszTimestamp, (const unsigned char*)pszTimestamp + strlen(pszTimestamp));
```

Comment:
IME it is usually considered bad style to use C-style casts in C++. They look less dangerous than they actually are. `reinterpret_cast` stands out more and is easier to grep for.

Reply:
Would be nice, although looking at our repo and upstream, `reinterpret_cast` is only really used by leveldb, almost no usage outside of that subsystem.